### PR TITLE
[OSDEV-913] Updating the submitted claim auto-reply message

### PR DESCRIPTION
**Pull request by vrwOAR**
Updating the text & html files of the claim_facility_submitted email files to align with current recommendations for account creation & to address current functionality that sends emails to the contact from the facility claim request form.

### CODE REVIEW

File: src/django/api/templates/mail/claim_facility_submitted_body.txt [hunk 1]
```diff
@@ -1,9 +1,16 @@
 {% block content %}
 Hi,
 
-You're receiving this email because we received your request to claim an Open Supply Hub facility. We'll review the request and be in touch soon.
+You're receiving this email because you were listed as the contact for a facility claim request on Open Supply Hub. We'll review the request and be in touch soon. The expected processing time for the initial review of a claim is 10 - 14 days. 
 
-Here is the information submitted with your request:
+Vendors can refer facilities to claim their profiles, however, the Facility Claim request will only be approved when it is submitted by a facility owner and/or senior management staff at the facility location in question.  If this claim is a referral, please visit our website [https://opensupplyhub.org/auth/register], and register as a user on Open Supply Hub. 
+
+To complete a claim request, you'll need to submit the following documentation:
```
Comment (vladsha-dev) [inline src/django/api/templates/mail/claim_facility_submitted_body.txt:8]: Looks like the same text should be added to the HTML template (`claim_facility_submitted_body.html`), doesn't it?
Sorry, if I missed smth.

**Review by vladsha-dev — CHANGES_REQUESTED** [review]
1. I left one comment.
2. There is one more file that needs to be updated: the RELEASE-NOTES.md file. Since these changes will not be included in the 1.10 release, please add a new chapter for version 1.11 to the RELEASE-NOTES.md file. You can find the template for the new version in the RELEASE-NOTES-TEMPLATE.md file.
After adding the new chapter, please update the "What's New" section with your updates. You can refer to examples of how this section is filled for other release versions.
I would write something like this:
[OSDEV-914](https://opensupplyhub.atlassian.net/browse/OSDEV-914) - Update the submitted claim auto-reply message to inform claimants about additional details regarding the expected timeline for review and proactive steps they can take to provide necessary documentation to the claims team.

**Review by killev — APPROVED** [review]

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Review by VadimKovalenkoSNF — APPROVED** [review]
lgtm

**Pull request merged**